LoopKit/InsulinKit/PumpEventType.swift

```diff
@@ -22,6 +22,7 @@ public enum PumpEventType: CaseIterable, Equatable {
     case rewind
     case suspend
     case tempBasal
+    case timeZoneSync(fromTimeZone: String, toTimeZone: String)
```

Comment:
The intrinsic components of a "time zone change" in Loop are the actual offsets, which are measured in seconds as an Int.  Loop doesn't really care about which TimeZone you're in, just the utc offset, and this is semantically important, as Loop doesn't drive schedules according to the timezone, but just the offset. You can stay in the same timezone, but when you cross certain dates, the offset changes.  And you could have multiple timezones with the same offset; when you switch from one to the other, your therapy schedule is not impacted in any way.

That's a lot of words to say maybe the "from" and "to" parts here should be Ints and the names be `fromGMTOffset` and `toGMTOffset`. 

The tidepool data model we need to upload in response to this is more of a "time change" event that has the "from" and "to" times as a localtime string, but the TidepoolService can calculate those two strings from the `date`  of the NewPumpEvent and the two offset ints.

Reply:
If we are capturing the iOS timezone change event, we should probably verify that it triggers during dst type events, where the timezone doesn't change, but the offset does.

Reply:
I also think we may need a third field to denote if this is a "system timezone change detected" vs a "user synced therapy schedules" event.